formula/src/main/java/com/instacart/formula/Evaluation.kt

```diff
@@ -1,9 +1,14 @@
 package com.instacart.formula
 
 /**
- * Represents the result of [Formula.evaluate].
+ * The result of [evaluate][Formula.evaluate] function.
+ *
+ * @param Output A data class returned by this formula that contains data and callbacks. When it is
```

Comment:
nit: it doesn't have to be a `data class`, right? Could be just a `class`?

Reply:
Adding `Usually` to the front :)